src/content/components/IntervalMarkerOverview.js

```diff
@@ -154,8 +167,17 @@ class IntervalMarkerOverview extends PureComponent {
                 onMouseDown={this._onMouseDown}
                 onMouseMove={this._onMouseMove}
                 onMouseUp={this._onMouseUp}
-                onMouseOut={this._onMouseOut}
-                title={title}/>
+                onMouseOut={this._onMouseOut}/>
+        {
+          tooltipContents
```

Comment:
we don't check if we're currently "selecting" here (like `isDragging` below). I think you can easily get this information with the boolean state `profileView.selection.isModifying`.